File: ABC/ABC259/B.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
    const double pai = 3.14159265359;
    double x, y, d;
    cin >> x >> y >> d;
    double ax, ay;
    double s = d * pai / 180;
    ax = x * cos(s) - y * sin(s);
    ay = x * sin(s) + y * cos(s);
    printf("%.10f %.10f\n", ax, ay);
}

File: ABC/ABC259/b.rs
use proconio::input;
use std::cmp::max;
use std::cmp::min;
use proconio::marker::Chars;
use std::f64::consts::PI;

fn main() {
    input!{
        a: f64,
        b: f64,
        d: f64,
    }

    let rad: f64 = d / 180. * PI;
    let x = a*rad.cos() - b*rad.sin();
    let y = a*rad.sin() + b*rad.cos();
    println!("{} {}", x, y);
}